process/transaction/interceptedTransaction.go

```diff
@@ -223,6 +232,36 @@ func isRelayedTx(funcName string) bool {
 		core.RelayedTransactionV2 == funcName
 }
 
+func isRelayedTxV3(tx *transaction.Transaction) bool {
```

Comment:
removed the method and reused the common one

the reason why I've added the extended check was to avoid treating txs that only have relayer address or relayer signature as relayed v3.. of course, these txs would have failed later